### ISSUE TITLE
Version files don't get created anymore

### ISSUE THREAD

**Issue by HaddockA** [CyclopsMC/IntegratedDynamics#757]
No issues with your mod, but I did get these two warnings in my log in MultiMC:

[08:55:25] [Client thread/WARN] [Integrated Crafting]: Could not get version info: java.io.FileNotFoundException: https://raw.githubusercontent.com/CyclopsMC/Versions/master/1.12/IntegratedCrafting.txt
[08:55:25] [Client thread/WARN] [Integrated Terminals]: Could not get version info: java.io.FileNotFoundException: https://raw.githubusercontent.com/CyclopsMC/Versions/master/1.12/IntegratedTerminals.txt

I pasted those URL's into my browser and got: "404: Not Found"

I'm assuming that it isn't something I should be too worried about.  Why do you actually need to connect to the internet, why can't you provide that info in your mod? ( Just out if interest. ;) )

**Comment by rubensworks** [comment]
Indeed, nothing to worry about. These files contain (read: *should* contain) version information about mods. Since recent changes to CurseForge, my automated process to generate these broke, which is why these don't exist or don't get updated anymore. I should look into this at some point...

**Comment by HaddockA** [comment]
Thank you for your help!

**Comment by rubensworks** [comment]
I'll leave this issue open as a reminder for myself to fix this at some point :-)

**Comment by rubensworks** [comment]
It looks like https://www.cfwidget.com/ (the way I get CurseForge data) is currently broken. Will have to revisit this when it is fixed.

Note to self when fixed:
Finish https://github.com/CyclopsMC/version-bot and add in the following hooks:

* hook1: https://github.com/CyclopsMC/Versions/
* hook2: infobooks
	-> https://github.com/CyclopsMC/infobook-html-integrateddynamics/blob/master/modpack.json
	-> https://github.com/CyclopsMC/infobook-html-evilcraft/blob/master/modpack.json

Run this every hour or so.

<blockquote><img src="/favicon.ico" width="48" align="right"><div><strong><a href="https://www.cfwidget.com/">CurseForge Widget & API</a></strong></div></blockquote>
<blockquote><img src="https://avatars2.githubusercontent.com/u/11808375?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicon.ico" height="14"> GitHub</div><div><strong><a href="https://github.com/CyclopsMC/version-bot">CyclopsMC/version-bot</a></strong></div><div>Detects new Cyclops mod versions. Contribute to CyclopsMC/version-bot development by creating an account on GitHub.</div></blockquote>
<blockquote><img src="https://avatars2.githubusercontent.com/u/11808375?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicon.ico" height="14"> GitHub</div><div><strong><a href="https://github.com/CyclopsMC/Versions">CyclopsMC/Versions</a></strong></div><div>Version repository for all Cyclops mods. Contribute to CyclopsMC/Versions development by creating an account on GitHub.</div></blockquote>
<blockquote><img src="https://avatars2.githubusercontent.com/u/11808375?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicon.ico" height="14"> GitHub</div><div><strong><a href="https://github.com/CyclopsMC/infobook-html-integrateddynamics">CyclopsMC/infobook-html-integrateddynamics</a></strong></div><div>Output the Integrated Dynamics infobook as HTML . Contribute to CyclopsMC/infobook-html-integrateddynamics development by creating an account on GitHub.</div></blockquote>
<blockquote><img src="https://avatars2.githubusercontent.com/u/11808375?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicon.ico" height="14"> GitHub</div><div><strong><a href="https://github.com/CyclopsMC/infobook-html-evilcraft">CyclopsMC/infobook-html-evilcraft</a></strong></div><div>Output the EvilCraft infobook as HTML . Contribute to CyclopsMC/infobook-html-evilcraft development by creating an account on GitHub.</div></blockquote>
